operator/v1/types_ingress.go

```diff
@@ -391,9 +391,10 @@ type LoadBalancerStrategy struct {
 	// Valid values are: Managed and Unmanaged.
 	//
 	// +kubebuilder:default:="Managed"
-	// +kubebuilder:validation:Optional
-	// +optional
-	DNSManagementPolicy LoadBalancerDNSManagementPolicy `json:"dnsManagementPolicy"`
+	// +kubebuilder:validation:Required
+	// +required
```

Comment:
Delete this line please, it doesn't serve any purpose as it isn't picked up by any generators. `+kubebuilder:validation:Required` is the correct tag